content/blog/cloudnative-pg-1-22.0-released/index.md

```diff
@@ -0,0 +1,134 @@
+parallel on-disk read/write operations. With CloudNativePG, users can define
+additional tablespace volumes, and also ensure they are used to temporary
+operations.
+
+This marks a pivotal step towards adopting Very Large Databases, building upon
+the foundation laid by Kubernetes volume snapshots introduced in version 1.21.
+
+Starting from version 1.22.0, the `ALTER SYSTEM` command is now disabled by
+default. This ensures that changes to the PostgreSQL configuration are
+orchestrated through the Kubernetes API. This streamlined approach guarantees
+coherence across the entire high-availability cluster and aligns with best
+practices for Infrastructure-as-Code.
+
+In terms of security, all supported versions now require TLS 1.3 for PostgreSQL
+connections by default, further bolstering the integrity of data transmission.
+
+Before initiating an upgrade, carefully review the
+[detailed instructions](https://cloudnative-pg.io/documentation/current/installation_upgrade/#upgrading-to-1220-1212-or-1205).
+New patch releases are now available for all supported versions, including
+1.21.2 and 1.20.5.
+
+Considering the notable changes introduced, we strongly recommend upgrading to
+CloudNativePG 1.22.0 at your earliest convenience. Alternatively, update to the
+latest patch version within your current minor release.
+
+With the release of 1.22.0, the 1.20.x minor version will reach its [end of
+life](https://cloudnative-pg.io/documentation/1.22/supported_releases/#support-status-of-cloudnativepg-releases)
+on January 21, 2023.
+
+This release addresses several bugs for improved stability. For a comprehensive list of changes, refer to the following release notes:
+
+- [Release notes for 1.22.0](https://cloudnative-pg.io/documentation/1.22/release_notes/v1.22/)
+- [Release notes for 1.21.2](https://cloudnative-pg.io/documentation/1.21/release_notes/v1.21/)
+- [Release notes for 1.20.5](https://cloudnative-pg.io/documentation/1.20/release_notes/v1.20/)
+
+Thank you for your continued support and engagement with CloudNativePG!
+
+
+
+The **CloudNativePG Community** has announced version 1.22.0, a new minor
+release of the **CloudNativePG Operator**, which introduces **declarative
+tablespaces** and **temporary tablespaces** along several enhancements and
+fixes.
+
+Tablespaces are a robust and widely embraced feature in database management
+systems that enhances the vertical scalability of a database, by decoupling the
+physical and logical modeling of data and by optimizing performance through
+parallel on-disk read/write operations. CloudNativePG enables you to define
+additional tablespace volumes, and control whether they can be used for
+temporary operations (by seamlessly managing the `temp_tablespaces` GUC).
+
+Tablespaces are another important step towards Very Large Database adoption,
+following the introduction of Kubernetes volume snapshots in 1.21.
+
+From 1.22, the `ALTER SYSTEM` command will be disabled by default, to ensure
```

Comment:
this and following ... it's a duplicate. @gbartolini are you experimenting different forms, or is this a copy-paste artifact?

Reply:
Mostly dupe. I added a bit of this one to the "first copy".
Will approve this PR but not merge it or publish it, leaving that to you @gbartolini .
The release documents are now published.